### ISSUE TITLE
Invalid date

### ISSUE THREAD

**Issue by madewild** [ec-doris/kohesio-backend#55]
On https://kohesio.eu/projects/Q11724 for instance we have "Start date: Invalid date". However it's not really an invalid date, just a missing value in the Wikibase. So we should put "Start date: Unknown" instead. I don't know if it's in the API or hardcoded in the UI?

**Comment by D063520** [comment]
The API response looks like:

curl 'https://kohesio.linkedopendata.eu/api/facet/eu/project?id=https://linkedopendata.eu/entity/Q11724' \
-X 'GET' \

{
    "countryLabel": "Czech Republic",
    "projectWebsite": "",
    "link": "https://linkedopendata.eu/entity/Q11724",
    "categoryLabel": "Investment in infrastructure, capacities and equipment in large companies directly linked to research and innovation activities",
    "description": "The aim of the project is to introduce an innovative production of mixed ammonium sulphate solution (so-called gouge), which is the basic raw material for the production of sulphur fertilisers (DASA 26 + 13S and DASA 25+ 12S). The output of the project will be product innovation (the innovative DASA product range) and process innovation (a new type of model slop reactor). The project builds on completed research of sulphur fertilisers with direct use of sulphuric acid carried out in cooperation with VUCHT a.s. a.",
    "videos": [],
    "regionUpper2": "ČESKÁ REPUBLIKA",
    "cofinancingRate": "25",
    "regionUpper3": "",
    "regionUpper1": "Severozápad",
    "objectiveLabel": "Research & Innovation",
    "countryCode": "CZ",
    "programmingPeriodLabel": "2014-2020",
    "programWebsite": "http://www.mpo.cz/cz/podnikani/dotace-a-podpora-podnikani/oppik-2014-2020/",
    "objectiveId": "TO01",
    "startTime": "",
    "euBudget": "2000000",
    "regionText": "Ústí nad Labem Region, Severozápad, ČESKÁ REPUBLIKA, ",
    "beneficiaries": [
        {
            "website": "http://www.lovochemie.cz/en/",
            "link": "https://linkedopendata.eu/entity/Q198964",
            "beneficiaryLabel": "Lovochemie",
            "wikidata": "http://wikidata.org/entity/Q15208257"
        }
    ],
    "budget": "8000000",
    "programLabel": "Enterprise and Innovation for Competitiveness",
    "item": "Q11724",
    "images": [],
    "geoJson": "{'type': 'Polygon', 'coordinates': [[[14.49122, 51.04353], [14.49945, 51.0461], [14.50586, 51.04389], [14.50745, 51.04105], [14.50145, 51.0317], [14.49936, 51.02235], [14.53308, 51.01587], [14.53908, 51.01111], [14.53376, 51.00605], [14.53503, 51.00382], [14.54278, 51.00477], [14.54759, 51.00536], [14.54987, 51.00535], [14.5557, 51.00711], [14.56038, 51.00729], [14.56114, 51.00842], [14.56384, 51.00955], [14.56871, 51.00456], [14.57819, 51.00018], [14.58254, 50.99307], [14.59879, 50.98624], [14.5998, 50.97857], [14.59638, 50.96933], [14.59279, 50.96702], [14.59575, 50.96248], [14.58627, 50.9527], [14.58094, 50.94241], [14.5714, 50.93694], [14.56066, 50.92537], [14.56487, 50.91869], [14.58301, 50.91472], [14.61739, 50.92522], [14.62583, 50.92562], [14.62837, 50.92574], [14.64732, 50.93153], [14.65177, 50.92665], [14.65257, 50.90624], [14.63751, 50.89861], [14.63368, 50.88728], [14.62583, 50.87482], [14.61943, 50.86466], [14.6188, 50.8578], [14.60731, 50.84298], [14.59563, 50.84011], [14.59027, 50.83814], [14.57134, 50.83896], [14.56707, 50.83666], [14.56946, 50.82961], [14.56422, 50.82593], [14.56395, 50.81868], [14.56021, 50.81427], [14.56274, 50.80719], [14.5606, 50.80008], [14.54077, 50.79483], [14.53488, 50.79654], [14.52557, 50.79557], [14.52264, 50.79809], [14.51005, 50.80113], [14.50598, 50.80547], [14.49474, 50.81078], [14.48415, 50.80988], [14.48323, 50.81462], [14.47608, 50.81631], [14.46037, 50.80999], [14.45124, 50.81381], [14.44397, 50.81147], [14.45573, 50.80597], [14.45186, 50.80327], [14.44969, 50.79599], [14.45823, 50.79324], [14.44894, 50.78636], [14.4573, 50.77854], [14.44863, 50.77097], [14.43357, 50.7728], [14.42741, 50.76585], [14.42196, 50.75138], [14.42193, 50.74976], [14.41839, 50.74837], [14.41813, 50.74131], [14.42946, 50.73599], [14.41767, 50.73177], [14.41172, 50.73628], [14.40343, 50.72675], [14.38932, 50.72283], [14.37624, 50.72011], [14.37305, 50.70939], [14.36823, 50.70428], [14.36784, 50.69865], [14.37637, 50.69373], [14.37778, 50.69056], [14.3734, 50.68668], [14.35157, 50.67714], [14.34355, 50.66792], [14.34305, 50.66651], [14.34276, 50.6602], [14.34259, 50.65792], [14.34548, 50.65753], [14.34755, 50.65664], [14.3514, 50.65606], [14.36246, 50.65685], [14.35987, 50.65059], [14.36616, 50.63686], [14.36032, 50.6287], [14.36785, 50.62417], [14.36846, 50.62079], [14.38055, 50.61135], [14.38115, 50.60105], [14.38623, 50.59864], [14.39254, 50.59113], [14.40417, 50.58758], [14.41128, 50.57798], [14.41418, 50.57406], [14.41822, 50.5569], [14.42559, 50.55363], [14.43684, 50.54072], [14.43972, 50.53982], [14.43764, 50.53546], [14.44209, 50.52917], [14.47031, 50.52665], [14.48649, 50.50503], [14.48306, 50.50644], [14.47893, 50.50649], [14.47256, 50.50922], [14.46946, 50.50928], [14.47283, 50.49951], [14.46162, 50.49841], [14.45588, 50.49573], [14.46964, 50.47927], [14.47175, 50.47314], [14.46911, 50.46898], [14.47369, 50.46014], [14.4721, 50.44975], [14.46161, 50.44947], [14.44859, 50.45355], [14.44902, 50.44644], [14.4252, 50.45103], [14.42528, 50.4403], [14.42002, 50.43407], [14.41225, 50.4313], [14.4203, 50.42788], [14.42053, 50.42328], [14.41638, 50.42025], [14.41128, 50.42164], [14.39054, 50.42726], [14.37797, 50.43547], [14.36642, 50.43183], [14.36023, 50.42859], [14.36836, 50.42423], [14.36996, 50.42124], [14.36765, 50.41846], [14.37642, 50.41051], [14.36698, 50.40963], [14.36321, 50.40628], [14.36924, 50.40319], [14.3704, 50.39899], [14.37915, 50.39541], [14.37894, 50.39341], [14.37678, 50.39302], [14.37702, 50.3886], [14.37753, 50.37593], [14.36745, 50.37707], [14.3673, 50.37436], [14.36902, 50.3712], [14.37194, 50.36001], [14.36734, 50.35235], [14.36021, 50.3502], [14.35554, 50.35258], [14.35205, 50.34965], [14.34254, 50.35459], [14.32873, 50.35759], [14.32315, 50.35623], [14.32265, 50.35251], [14.30846, 50.35475], [14.30748, 50.35032], [14.30102, 50.35273], [14.29686, 50.35137], [14.29463, 50.3495], [14.28963, 50.34953], [14.27421, 50.35384], [14.25371, 50.33423], [14.25081, 50.32876], [14.24746, 50.32846], [14.24751, 50.3297], [14.24863, 50.33061], [14.24024, 50.3321], [14.24053, 50.33354], [14.23135, 50.33526], [14.22123, 50.33875], [14.2156, 50.33994], [14.21135, 50.33514], [14.20959, 50.3354], [14.20189, 50.33677], [14.20047, 50.33544], [14.19757, 50.33746], [14.19328, 50.3393], [14.19762, 50.34296], [14.19337, 50.34395], [14.18225, 50.34527], [14.18148, 50.33516], [14.16872, 50.3343], [14.14311, 50.33085], [14.14066, 50.33102], [14.13916, 50.33266], [14.127, 50.33277], [14.12522, 50.33301], [14.11168, 50.33485], [14.10162, 50.33434], [14.09662, 50.33409], [14.07788, 50.33479], [14.06537, 50.34033], [14.05165, 50.34129], [14.03723, 50.34533], [14.03014, 50.34071], [14.01995, 50.34786], [14.00824, 50.3463], [13.99797, 50.34195], [13.99154, 50.33575], [13.98203, 50.33321], [13.98358, 50.32334], [13.99116, 50.31704], [13.9886, 50.31051], [13.99195, 50.30453], [13.97286, 50.30193], [13.96972, 50.30421], [13.96122, 50.30394], [13.95223, 50.30703], [13.95115, 50.30427], [13.9563, 50.30027], [13.95687, 50.2928], [13.95003, 50.28836], [13.94512, 50.28809], [13.94412, 50.288], [13.94059, 50.28086], [13.9325, 50.2788], [13.9299, 50.27833], [13.92741, 50.27839], [13.92628, 50.27695], [13.92505, 50.27509], [13.91853, 50.27249], [13.91648, 50.27245], [13.91164, 50.27141], [13.91336, 50.2745], [13.91062, 50.27561], [13.87906, 50.27009], [13.8732, 50.26313], [13.86555, 50.25963], [13.86755, 50.25505], [13.8641, 50.25174], [13.86386, 50.24419], [13.85664, 50.24238], [13.8522, 50.24222], [13.84663, 50.24201], [13.84312, 50.24156], [13.83947, 50.24001], [13.83839, 50.24114], [13.83773, 50.24222], [13.82674, 50.26021], [13.82026, 50.25947], [13.81531, 50.26263], [13.79972, 50.26005], [13.79511, 50.25236], [13.79069, 50.24949], [13.78773, 50.24701], [13.7816, 50.24222], [13.77678, 50.23619], [13.76475, 50.23259], [13.76095, 50.23944], [13.75609, 50.23432], [13.75351, 50.23488], [13.74992, 50.22361], [13.72084, 50.22542], [13.69965, 50.22115], [13.68235, 50.22583], [13.67385, 50.22416], [13.67165, 50.22372], [13.66983, 50.23817], [13.6645, 50.23565], [13.65597, 50.22851], [13.65249, 50.22559], [13.65055, 50.22336], [13.64486, 50.21854], [13.63855, 50.21683], [13.632, 50.20866], [13.61614, 50.21085], [13.61858, 50.20855], [13.62089, 50.20634], [13.6224, 50.20422], [13.62292, 50.20251], [13.60698, 50.19557], [13.60652, 50.19244], [13.59985, 50.19365], [13.5912, 50.18921], [13.56966, 50.18546], [13.56052, 50.19148], [13.54875, 50.19089], [13.53577, 50.18898], [13.53391, 50.18391], [13.5159, 50.17777], [13.5164, 50.1737], [13.51219, 50.16849], [13.50435, 50.16601], [13.48937, 50.16553], [13.4885, 50.15752], [13.49369, 50.14671], [13.49079, 50.14439], [13.47931, 50.14331], [13.48695, 50.14184], [13.48956, 50.13527], [13.49212, 50.12797], [13.48789, 50.12643], [13.47908, 50.12687], [13.47061, 50.13151], [13.46767, 50.12623], [13.47027, 50.11419], [13.46094, 50.11296], [13.46371, 50.10955], [13.46254, 50.10607], [13.45597, 50.10454], [13.44216, 50.10753], [13.44057, 50.10913], [13.42668, 50.10908], [13.42499, 50.10416], [13.41666, 50.09995], [13.41177, 50.1004], [13.40564, 50.09779], [13.4093, 50.09464], [13.40705, 50.08962], [13.40385, 50.08552], [13.39426, 50.08078], [13.39013, 50.08238], [13.38195, 50.07977], [13.37574, 50.07799], [13.36875, 50.07965], [13.37016, 50.09055], [13.36551, 50.09081], [13.35701, 50.09474], [13.35769, 50.09777], [13.35438, 50.10042], [13.35708, 50.10285], [13.35497, 50.10386], [13.34352, 50.09938], [13.34008, 50.09263], [13.33374, 50.08741], [13.32518, 50.08791], [13.32435, 50.10336], [13.31949, 50.10113], [13.30137, 50.09964], [13.29768, 50.10141], [13.2939, 50.11328], [13.28595, 50.11579], [13.28684, 50.11825], [13.28034, 50.12097], [13.27396, 50.12721], [13.26602, 50.12813], [13.2461, 50.12608], [13.23354, 50.1307], [13.23, 50.13331], [13.24054, 50.1353], [13.23653, 50.13943], [13.24842, 50.14317], [13.25079, 50.1464], [13.25979, 50.14295], [13.26833, 50.14281], [13.27667, 50.13784], [13.28126, 50.13769], [13.28349, 50.14438], [13.27785, 50.14931], [13.2681, 50.15259], [13.26637, 50.15731], [13.27339, 50.15882], [13.2841, 50.15658], [13.28598, 50.16745], [13.27501, 50.16897], [13.2788, 50.17678], [13.27452, 50.18481], [13.26491, 50.19201], [13.25011, 50.19533], [13.25188, 50.19928], [13.2494, 50.2047], [13.24232, 50.20746], [13.23955, 50.21127], [13.21246, 50.21711], [13.2148, 50.21936], [13.22673, 50.21917], [13.24039, 50.22282], [13.25104, 50.222], [13.25033, 50.22506], [13.25394, 50.22932], [13.24324, 50.23301], [13.23215, 50.23273], [13.22656, 50.23602], [13.22297, 50.24222], [13.22574, 50.24528], [13.23617, 50.24769], [13.23513, 50.25493], [13.23066, 50.27116], [13.21834, 50.27442], [13.21758, 50.28046], [13.21002, 50.28225], [13.22008, 50.28636], [13.23205, 50.29814], [13.23204, 50.3017], [13.22531, 50.30738], [13.22332, 50.31238], [13.23734, 50.32523], [13.23348, 50.33173], [13.23636, 50.33626], [13.22869, 50.33905], [13.22189, 50.35226], [13.20908, 50.35474], [13.2106, 50.35869], [13.2083, 50.36081], [13.19358, 50.35371], [13.16374, 50.35337], [13.15877, 50.35292], [13.14838, 50.35744], [13.13376, 50.35956], [13.12129, 50.35393], [13.11575, 50.34618], [13.10944, 50.35353], [13.09853, 50.35658], [13.09322, 50.35487], [13.10049, 50.36832], [13.10842, 50.37045], [13.09726, 50.3821], [13.08021, 50.38118], [13.06984, 50.38909], [13.06626, 50.38695], [13.04838, 50.38852], [13.04345, 50.38463], [13.04182, 50.3852], [13.02616, 50.39069], [13.02132, 50.39576], [13.01751, 50.39487], [13.0113, 50.39369], [12.99778, 50.39119], [12.98686, 50.39289], [12.97437, 50.38938], [12.9717, 50.39261], [12.96984, 50.39659], [12.96863, 50.39808], [12.96665, 50.39906], [12.964, 50.40101], [12.95821, 50.40185], [12.95194, 50.4034], [12.94814, 50.40431], [12.9471, 50.40648], [12.94102, 50.40834], [12.94272, 50.4114], [12.98198, 50.41926], [12.99335, 50.43045], [12.99838, 50.43505], [13.01694, 50.44441], [13.02223, 50.4514], [13.019, 50.4666], [13.02375, 50.48828], [13.03177, 50.5004], [13.03169, 50.50884], [13.04182, 50.51039], [13.04407, 50.51073], [13.04597, 50.50977], [13.05079, 50.50733], [13.05177, 50.50586], [13.05325, 50.50338], [13.05598, 50.50126], [13.05946, 50.50105], [13.08308, 50.50024], [13.10137, 50.50313], [13.13102, 50.51791], [13.13774, 50.51251], [13.13899, 50.5066], [13.15432, 50.50824], [13.17591, 50.50386], [13.18609, 50.50371], [13.19336, 50.50317], [13.19638, 50.50547], [13.19553, 50.51569], [13.20738, 50.52217], [13.21019, 50.53015], [13.22063, 50.54067], [13.22232, 50.54587], [13.22856, 50.54988], [13.2238, 50.56366], [13.23604, 50.57143], [13.23423, 50.57864], [13.25441, 50.59488], [13.26075, 50.59262], [13.28035, 50.59205], [13.2915, 50.5757], [13.3014, 50.57884], [13.32208, 50.58135], [13.32467, 50.58538], [13.3194, 50.60217], [13.32504, 50.60465], [13.32615, 50.60795], [13.33373, 50.60739], [13.33875, 50.61233], [13.35214, 50.61463], [13.36087, 50.61864], [13.36426, 50.6181], [13.37715, 50.62833], [13.37175, 50.64992], [13.3756, 50.64959], [13.38195, 50.64526], [13.39121, 50.64594], [13.39291, 50.64215], [13.40537, 50.63372], [13.41363, 50.62032], [13.41481, 50.61709], [13.42086, 50.61599], [13.42547, 50.61645], [13.42837, 50.611], [13.44226, 50.61051], [13.46272, 50.60183], [13.47923, 50.61646], [13.49303, 50.62409], [13.49754, 50.63129], [13.50185, 50.63364], [13.51838, 50.63699], [13.52355, 50.63958], [13.52436, 50.64788], [13.51611, 50.64939], [13.5138, 50.65502], [13.52928, 50.66692], [13.54057, 50.6696], [13.53918, 50.67353], [13.54372, 50.67791], [13.54088, 50.68818], [13.52572, 50.70307], [13.54029, 50.70815], [13.5561, 50.71439], [13.5866, 50.71295], [13.60225, 50.71006], [13.62519, 50.71552], [13.62612, 50.72201], [13.64294, 50.7292], [13.65217, 50.73036], [13.65893, 50.73065], [13.66731, 50.73129], [13.67385, 50.72769], [13.6885, 50.71962], [13.70737, 50.71699], [13.71201, 50.72002], [13.70955, 50.72506], [13.71478, 50.7259], [13.72811, 50.73319], [13.73942, 50.72777], [13.74677, 50.72834], [13.75802, 50.73506], [13.76961, 50.73338], [13.77825, 50.73582], [13.79139, 50.73309], [13.80666, 50.73296], [13.82591, 50.72462], [13.8349, 50.72652], [13.84548, 50.72561], [13.8569, 50.72843], [13.86295, 50.74263], [13.87909, 50.73833], [13.8843, 50.73908], [13.89849, 50.74495], [13.90163, 50.75315], [13.88986, 50.76307], [13.88791, 50.76967], [13.89924, 50.78473], [13.89651, 50.7892], [13.89836, 50.79215], [13.90347, 50.79353], [13.92487, 50.7888], [13.9402, 50.79055], [13.95486, 50.8004], [13.95565, 50.80812], [13.97149, 50.81159], [13.99004, 50.81917], [14.00162, 50.81506], [14.00346, 50.81027], [14.01671, 50.81097], [14.03068, 50.80443], [14.03523, 50.80549], [14.03962, 50.81036], [14.04844, 50.81018], [14.07838, 50.81241], [14.07917, 50.81831], [14.08852, 50.82536], [14.10162, 50.82814], [14.12522, 50.83314], [14.12965, 50.83408], [14.13287, 50.83399], [14.13685, 50.83606], [14.1401, 50.83817], [14.14489, 50.84088], [14.14922, 50.84233], [14.16193, 50.84717], [14.2019, 50.85171], [14.21644, 50.85988], [14.22329, 50.85924], [14.23558, 50.87554], [14.23637, 50.8815], [14.23315, 50.88752], [14.25037, 50.88755], [14.2696, 50.89519], [14.28167, 50.8924], [14.2924, 50.88532], [14.3045, 50.88423], [14.31285, 50.887], [14.31936, 50.88874], [14.32603, 50.8896], [14.34808, 50.89312], [14.34942, 50.89979], [14.37251, 50.89657], [14.38834, 50.9003], [14.38692, 50.91597], [14.40214, 50.92407], [14.40012, 50.93355], [14.39673, 50.93649], [14.38785, 50.94101], [14.37307, 50.93864], [14.36877, 50.94186], [14.36162, 50.9418], [14.34371, 50.94894], [14.32571, 50.94978], [14.3124, 50.95372], [14.31662, 50.96], [14.3038, 50.96561], [14.31445, 50.97257], [14.32739, 50.97317], [14.32647, 50.97938], [14.32999, 50.9824], [14.32181, 50.98514], [14.30916, 50.98355], [14.29811, 50.97913], [14.28459, 50.97741], [14.27612, 50.98258], [14.25853, 50.98799], [14.2591, 50.99212], [14.26758, 50.99994], [14.26137, 51.00819], [14.26751, 51.00908], [14.28197, 51.01559], [14.27864, 51.01911], [14.28562, 51.0262], [14.2855, 51.03095], [14.27416, 51.03907], [14.28834, 51.03942], [14.30141, 51.05442], [14.31787, 51.0547], [14.33499, 51.04007], [14.34494, 51.03908], [14.34678, 51.04112], [14.35097, 51.04272], [14.35352, 51.04326], [14.35649, 51.04313], [14.36199, 51.04543], [14.37359, 51.03942], [14.38192, 51.03822], [14.38373, 51.02864], [14.39161, 51.02366], [14.39591, 51.02237], [14.40058, 51.02209], [14.40532, 51.02083], [14.41128, 51.01897], [14.4216, 51.01973], [14.45429, 51.03562], [14.46564, 51.03474], [14.47462, 51.02931], [14.4762, 51.02539], [14.49141, 51.02371], [14.49396, 51.03589], [14.49122, 51.04353]]]}",
    "coordinates": [
        "Point(14.0428 50.4808)",
        "Point(14.0428 50.4808)",
        "Point(14.0428 50.4808)",
        "Point(14.0428 50.4808)",
        "Point(14.0428 50.4808)",
        "Point(14.0428 50.4808)",
        "Point(14.0428 50.4808)",
        "Point(14.0428 50.4808)"
    ],
    "label": "Innovation of blending technology for the preparation of sulphur fertilisers",
    "managingAuthorityLabel": "Ministerstvo průmyslu a obchodu  / Útvar náměstka ministra pro fondy EU",
    "fundLabel": "European Regional Development Fund",
    "endTime": "2020-06-30",
    "region": "Ústí nad Labem Region"
}

So basically it does not provide a start date because we do not have one. The invalid date is coming from the front-end. 
@raphdom should we fix it in the front-end or do you prefer that I put in the backend "unknown"?
